extensions/2.0/Vendor/EXT_mesh_gpu_instancing/README.md

```diff
@@ -57,6 +57,8 @@ All attribute accessors in a given node **must** have the same `count`.
 
 > **Implementation Note:** When instancing is used on the node, the non-instanced version of the mesh should not be rendered.
 
+Because the extension does not provide a way to specify a non-instanced fallback, files that use the extension should specify it in `extensionsRequired` array.
```

Comment:
Please use formatting for "should" here.

Reply:
Updated, thanks!